integration_tests/cairozero_test.go

```diff
@@ -321,6 +323,8 @@ func runVm(path string) (time.Duration, string, string, string, error) {
 	layout := "plain"
 	if strings.Contains(path, ".small") {
 		layout = "small"
+	} else if strings.Contains(path, ".starknet_without_keccak") {
+		layout = "starknet"
 	} else if strings.Contains(path, ".starknet_with_keccak") {
 		layout = "starknet_with_keccak"
 	}
```

Comment:
You can just check `.starknet_with_keccak` first and then check for `.starknet` to avoid conflicts. You don't need the `_without_keccak` suffix.

Reply:
true! wanted to respect rust vm order but it strictly doesnt matter, will do that!